src/dsp_tools/commands/ingest_xmlupload/user_information.py

```diff
@@ -0,0 +1,137 @@
+from dataclasses import dataclass
+from pathlib import Path
+from typing import Any, Protocol
+
+import pandas as pd
+
+# pylint: disable=too-few-public-methods
+
+
+separator = "\n    "
+list_separator = "\n    - "
+
+maximum_prints = 20
+
+csv_filepath = Path.home()
+unused_media_filename = "UnusedMediaUploadedInSipi.csv"
+no_uuid_filename = "NotUploadedFilesToSipi.csv"
+
+
+class UserInformation(Protocol):
```

Comment:
I started a protocol because of the assumption that other errors will join that will also be of the same protocol. But until that is the case we can delete it.